### ISSUE TITLE
New projects not functional

### ISSUE THREAD

**Issue by xoorath** [SpringRoll/SpringRoll#130]
Been a while since I've played with springroll, but downloading the latest springroll studio and starting a new project, as soon as you go to "Tasks" we get the following.

![image](https://cloud.githubusercontent.com/assets/769014/22338659/e81c88e6-e3b5-11e6-839b-4da4446875b7.png)

My guess here is bad just versioning practices, using `"grunt": "*",` in package.json, but I haven't figured out what version we *really* want here, so I haven't found a workaround just yet.
```
grunt
>> Local Npm module "grunt-bower-task" not found. Is it installed?
>> Local Npm module "grunt-concat-json" not found. Is it installed?
>> Local Npm module "grunt-contrib-clean" not found. Is it installed?
>> Local Npm module "grunt-contrib-concat" not found. Is it installed?
>> Local Npm module "grunt-contrib-connect" not found. Is it installed?
>> Local Npm module "grunt-contrib-copy" not found. Is it installed?
>> Local Npm module "grunt-contrib-jshint" not found. Is it installed?
>> Local Npm module "grunt-contrib-less" not found. Is it installed?
>> Local Npm module "grunt-contrib-uglify" not found. Is it installed?
>> Local Npm module "grunt-contrib-watch" not found. Is it installed?
>> Local Npm module "grunt-newer" not found. Is it installed?
>> Local Npm module "grunt-simple-version" not found. Is it installed?
>> Local Npm module "grunt-springroll-helper" not found. Is it installed?
>> Local Npm module "grunt-text-replace" not found. Is it installed?
Warning: Task "clean:main" not found. Use --force to continue.

Aborted due to warnings.
```

**Comment by yanshcherbakovatredspace** [comment]
This is a known bug in Springroll Studio, see: https://github.com/SpringRoll/SpringRollStudio/issues/10

To resolve, you may follow the workaround states in that issue, or you may also simply run the following commands manually:

```
npm install grunt-springroll-helper
grunt _springroll_usertasks
```

**Comment by chipbell4** [comment]
Okay, this should now be resolved with https://github.com/SpringRoll/grunt-springroll/pull/6 and is available in [grunt-springroll 0.7.11](https://www.npmjs.com/package/grunt-springroll). Let me know if you're still running into this issue
